exercises/binary/src/example.c

```diff
@@ -0,0 +1,23 @@
+int len(char *str)
+{
+   int count = 0;
+   while (*str++) {
+      count++;
+   }
+   return count;
+}
+
+int convert(char *input)
+{
+   int input_len = len(input);
```

Comment:
You should use the standard lib function `strlen()` instead of writing your own.

That also means that the variables `input_length` and `i` need to be of type `size_t`